blocks/core/ff_module/ff_module-date-stepper/ff_module-date-stepper.xsl

```diff
@@ -3,15 +3,7 @@
     <div class="ff_module-date-stepper">
         <a class="ff_module-date-stepper__link ff_module-date-stepper__link--prev" href="{$data//planner/@previousUrl}"><span class="ff_icon ff_icon-page-back-blue"></span><span class="ff_util-icon-text">Previous</span></a>
         <h3 class="ff_module-date-stepper__title">
-            <xsl:choose>
-                <xsl:when test="$data//planner/@type = 'day'">
-                    <span class="ff_module-date-stepper__title--type">Today</span>
-                </xsl:when>
-                <xsl:when test="$data//planner/@type = 'week'">
-                    <span class="ff_module-date-stepper__title--type">Week</span>
-                </xsl:when>
-            </xsl:choose>
-            <span class="ff_module-date-stepper__title--date"><xsl:value-of select="$data//planner/@date"/></span>
+            <span class="ff_module-date-stepper__title"><xsl:value-of select="$data//planner/@title"/></span>
```

Comment:
@bwfox This still needs to be two separate items as the design calls for the day-name to be bold. Se we need to retain the `--type` and `--date` `<span>`'s